src/React.AspNet/BabelFileMiddleware.cs

```diff
@@ -73,22 +74,22 @@ public BabelFileMiddleware(RequestDelegate next, BabelFileOptions options, IHost
 		/// <returns></returns>
 		private StaticFileMiddleware CreateFileMiddleware(IBabel babel)
 		{
-			return new StaticFileMiddleware(
+            return new StaticFileMiddleware(
 				_next,
 				_hostingEnv,
-				new StaticFileOptions
-				{
-					ContentTypeProvider = _options.StaticFileOptions.ContentTypeProvider,
-					DefaultContentType = _options.StaticFileOptions.DefaultContentType,
-					OnPrepareResponse = _options.StaticFileOptions.OnPrepareResponse,
-					RequestPath = _options.StaticFileOptions.RequestPath,
-					ServeUnknownFileTypes = _options.StaticFileOptions.ServeUnknownFileTypes,
-					FileProvider = new BabelFileSystem(
-						babel, 
-						_options.StaticFileOptions.FileProvider ?? _hostingEnv.WebRootFileProvider,
-						_options.Extensions
-					)
-				},
+                Options.Create(new StaticFileOptions
```

Comment:
Could you please adjust the spacing here? It should be using tabs rather than spaces. I though there was an Editorconfig file in the repo but it looks like I forgot to commit it.